File: 0070-climbing-stairs/0070-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
    int ans1=0;
    vector<int>ans;
    ans.push_back(0);
    ans.push_back(1);
       for(int i=1;i<=n;i++){
       int first=ans[i-1];
       int second=ans[i];
       ans1=(first+second);
       ans.push_back(ans1);
       }
       return ans[ans.size()-1];
    }
};

File: 0070-climbing-stairs/0070-climbing-stairs.py
class Solution:
    def climbStairs(self, n: int) -> int:
        ans1 = 0
        ans = [0, 1]
        for i in range(1, n + 1):
            first = ans[i - 1]
            second = ans[i]
            ans1 = first + second
            ans.append(ans1)
        return ans[-1]
